web/packages/teleterm/src/ui/Search/SearchContext.test.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+
+import { SearchContextProvider, useSearchContext } from './SearchContext';
+
+describe('lockOpen', () => {
```

Comment:
These tests are a bit ewww but it's all due to how we have to manipulate the resolution status of the promise passed to `lockOpen`.

I guess I could at least make it a table test as the body of these two is essentially the same, one just resolves the promise passed to `lockOpen` and the other rejects it.

Reply:
+1 for the table test.